generators/app/templates/src/utils/validations/validateUtils.ts

```diff
@@ -4,27 +4,37 @@ import './i18n';
 
 // REGEXS
 const emailRegex = /^(([^<>()·=~ºªÇ¨?¿*^|#¢∞¬÷"$%"≠´}{![\]\\.,;:\s@"]+(\.[^<>·$%&/=~ºªÇ¨?¿*^|#¢∞¬÷""≠´}{!()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-const alphanumericRegex = /^[a-zA-Z0-9]*$/g;
+const alphanumericRegex = /^[a-záéíóúäëïºöüA-ZÁÉÍÓÚÄËÏÖÜ0-9 ]*$/g;
 const onlyTextRegex = /^[a-zA-Z\s]*$/;
 const onlyNumberRegex = /^[0-9]*$/g;
 
 // VALIDATIONS
-export const validateRequired = { required: i18next.t('VALIDATIONS:REQUIRED_FIELD') };
+export const validateRequired = {
+  required: {
+    value: true,
+    message: i18next.t('VALIDATIONS:REQUIRED_FIELD')
```

Comment:
Doesn't this and `validateEmail` need `as string` too?

Reply:
I think not because both validations return a message as a string. The problem is not the i18n, it's the 
```
validate: (value: string) => boolean || i18n
```
TS is ok with this